### DISCUSSION TITLE
Saved new ArchR project but new ArchR object has old output directory

### DISCUSSION THREAD

**Discussion by zrcjessica** [GreenleafLab/ArchR#1222] [Questions / Documentation]
Hello,

I performed some filtering on my ArchR project and added some metadata, and then saved it as a new ArchR object with a new output directory. In my log file, it does say that the new arrow files are being written to the output directory I specfied (`save-cocaine_with_naive-FILT`):

```
Copying ArchRProject to new outputDirectory : save-cocaine_with_naive-FILT
Copying Arrow Files...
Copying Arrow Files (1 of 12)
Copying Arrow Files (2 of 12)
Copying Arrow Files (3 of 12)
Copying Arrow Files (4 of 12)
Copying Arrow Files (5 of 12)
Copying Arrow Files (6 of 12)
Copying Arrow Files (7 of 12)
Copying Arrow Files (8 of 12)
Copying Arrow Files (9 of 12)
Copying Arrow Files (10 of 12)
Copying Arrow Files (11 of 12)
Copying Arrow Files (12 of 12)
```

I then loaded the `Save-ArchR-Project.rds` file but found that the output directory associated with the new ArchR project was the same as the old one:

```
outputDirectory: cocaine_with_naive 
```

I saved the new ArchR project with the following command:

```r
saveArchRProject(ArchRProj = proj.filt, 
	outputDirectory = save-cocaine_with_naive-FILT, load = FALSE)
```

What did I do wrong? It seems that all my updates are updating the original files associated with the original ArchR project.

Thanks for your help!

UPDATE: I tried setting `load=TRUE` but I still get the same issue when I load the .rds file in a new session - the output directory is associated with the original output directory, and when I try to save PDFs with the new ArchR project, they save to the original output directory, not the output directory associated with the .rds file that stores the current ArchR project.

**Comment by Feilijiang** [comment]
`proj.filt<-saveArchRProject(ArchRProj = proj.filt, outputDirectory = save-cocaine_with_naive-FILT, load = TRUE)`
Hope it helps.
